game/scripts/vscripts/TutorialGraph/Steps.ts

```diff
@@ -3,13 +3,24 @@ import * as dg from "../Dialog"
 import * as tg from "./Core"
 import { getSoundDuration } from "../Sounds"
 
-const isHeroNearby = (location: Vector, radius: number) => FindUnitsInRadius(
-    DotaTeam.BADGUYS, location, undefined, radius,
-    UnitTargetTeam.BOTH,
-    UnitTargetType.HERO,
-    UnitTargetFlags.INVULNERABLE + UnitTargetFlags.OUT_OF_WORLD + UnitTargetFlags.MAGIC_IMMUNE_ENEMIES,
-    0, false
-).length > 0
+const isHeroNearby = (location: Vector, radius: number, heroName: string) => {
```

Comment:
This should really just check that `getPlayerHero` is near the location within radius. When I added this i didn't intend for this to be used in any other way but we didn't have the function for getting the player hero yet so I made something messy. Perhaps rename to `isPlayerHeroNearby` then.